src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrise = 40;
+  const discontSevenDay = 50;
+  const discontThreeDay = 20;
+
+  if (days >= 7) {
```

Comment:
Consider using constants for the number of days that define the thresholds for discounts (3 and 7 days) to avoid hardcoding these values directly in the expressions, as per the checklist.